app/src/foss/kotlin/com/wire/android/navigation/TrackingNavController.kt

```diff
@@ -0,0 +1,8 @@
+package com.wire.android.navigation
+
+import androidx.compose.runtime.Composable
+import androidx.navigation.compose.rememberNavController
+
+@Composable
+fun rememberTrackingAnimatedNavController(nameFromRoute: (String) -> String?) =
+    rememberNavController()
```

Comment:
I did get `conflicting declaration` error with this fix, I assume this is related to the fact that the same function is declared in prod src set, I did try it without this change and it works now 🤔 i will assume the error i had before had to do with gradle cache